static/js/stat_var_hierarchy/stat_var_search.tsx

```diff
@@ -107,27 +114,31 @@ export class StatVarHierarchySearch extends React.Component<
         </div>
         {renderResults && (
           <div className="statvar-hierarchy-search-results" tabIndex={-1}>
+            {showResultCount && (
+              <div className="result-count-message">
```

Comment:
We could use the same "gray" for this, since this is not a clickable result and is more of a hint.

Reply:
done